MEAnalysis/python/MakeDatasetFiles.py

```diff
@@ -166,7 +166,7 @@
             #merge lumi files
 
         #merge lumi files
-        total_lumis = lumis[1]
+        total_lumis = LumiList()
```

Comment:
I think it was a bug. I think it double counts lumis[1].
Anyway, it crashes if you run with only one file